src/webui/api/torrentscontroller.cpp

```diff
@@ -196,6 +196,34 @@ namespace
 
         return QVariantList {dht, pex, lsd};
     }
+
+    // TODO this is a duplicate of PeersAdditionDialog::parsePeer(). move this to Utils::Net
```

Comment:
It should be:
```c++
// TODO: This is a duplicate of PeersAdditionDialog::parsePeer(). Move this to Utils::Net.
```
Then it will be correctly handled by IDEs (that support TODO entries).